README.md

```diff
@@ -111,48 +110,18 @@ your [development environment](https://hyperledger-fabric.readthedocs.io/en/rele
 
 Moreover, we assume that you are familiar with the Intel SGX SDK.
 
-### Requirements
-
-Make sure that you have the following required dependencies installed
-(see also below [docker section](#docker) for a _pre-configured fpc developer
-docker image_ which handles the corresponding installs automatically
-for you):
-
-* Linux (OS) (we recommend Ubuntu 18.04, see [list](https://github.com/intel/linux-sgx#prerequisites) supported OS)
-
-* CMake v3.5.1 or higher
-
-* [Go](https://golang.org/) 1.11.x or higher
-
-* Docker 18.x
-
-* Protocol Buffers 3.x and [Nanopb](http://github.com/nanopb/nanopb) 0.3.9.2
-
-* SGX SDK v2.6 for [Linux](https://github.com/intel/linux-sgx)
+There are 2 different ways to develop Fabric Private Chaincode. Using our preconfigured Docker container development environment or setting up your local system with all required software dependencies to build and develop chaincode locally. 
 
-* Credentials for Intel Attestation Service, read [here](#intel-attestation-service-ias) (for hardware-mode SGX)
-
-* [SSL](https://github.com/intel/intel-sgx-ssl)  for SGX SDK v2.4.1 (we recommend using OpenSSL 1.1.0j) 
-
-* Hyperledger [Fabric](https://github.com/hyperledger/fabric) v1.4.3
-
-* Clang-format 6.x or higher
-
-* [PlantUML](http://plantuml.com/) including Graphviz (for building documentation only) 
-
-
-#### Docker
+### **Option 1: Using the Docker-based FPC Development Environment**
```

Comment:
why are we making titles also bold? i think we should rely on the formatting of headers by markdown.  That said, the headers do not distinguish by much. But in that case, we might more consider whether we couldn't flatten the hiearchacy?  Or is there a way to give (automatic) section numbers in markdown? that also would make it more clear ..

Reply:
We used bold to set these headers apart at this level.  The hierarchy is getting too deep otherwise and the headers start look smaller than the body text.

Reply:
hmm, but this would rather ask for a change of hierachy rather than change (some) headers of a certain level from their standard formatting?

Reply:
an alternative could of course be to maybe move option 2 to a separate document with the assumption many folks would start right away with docker and wouldn't care "bare metal" (and others might just replicate the commands from the docker file rather than read the instructions :-)